management_api_app/db/repositories/resource_templates.py

```diff
@@ -51,6 +51,16 @@ def get_current_template(self, template_name: str, resource_type: ResourceType)
             raise EntityDoesNotExist
         return parse_obj_as(ResourceTemplate, templates[0])
 
+    def get_current_user_resource_template(self, user_resource_template_name, parent_service_name) -> UserResourceTemplate:
+        """
+        Returns full user_resource_template for the current version of the template_name, parent_service_template_name combo
+        """
+        query = self._template_by_name_query(user_resource_template_name, ResourceType.UserResource) + f' AND c.parentWorkspaceService = "{parent_service_name}" AND c.current = true'
+        templates = self.query(query=query)
+        if len(templates) != 1:
```

Comment:
Should we handle cases where l**en(templates) == 0** and **len(templates) > 1** separately?

Reply:
Good point @msebragge

In reality - anything above 1 should never happen (as we only have one current) but never is a dangerous word :) will split them up so we have a good troubleshooting path, in case it happens